packages/react/src/primitives/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {
+  Root,
+  List,
+  Trigger as RadixTab,
+  Content as Panel,
```

Comment:
Perhaps this renaming isn't necessary. It just helped me in thinking through how to use Radix.

Reply:
Makes sense to me